src/vde_backend/compareConfig.py

```diff
@@ -0,0 +1,126 @@
+__title__ = "preprocessFluksoSensors"
+__version__ = "1.0.0"
+__author__ = "Alexandre Heneffe"
+__license__ = "MIT"
+
+
+"""
+Script to compare 2 specific config files : 
+- 2 arguments : 2 config excel file paths of same format
+- file path name format : 'ConfigurationNN_YYYYmmdd.xlsx'
+- Show which home ids are new, which sensors ids are new
+"""
+
+# standard library
+import argparse
+import sys
+
+# 3rd party packages
+import pandas as pd
+
+# local source
+import pyToCassandra as ptc
+from sensorConfig import Configuration
+from constants import CASSANDRA_KEYSPACE
+from utils import getLastRegisteredConfig
+from preprocessFluksoSensors import getCompactSensorDF
+
+
+
+def compare_configs(old_config, new_config):
+	""" 
+	Go through the 2 config home ids and sensors ids
+	and detect new changes
+	"""
+
+	print("Old config : ")
+	print(old_config)
+	print("New config : ")
+	print(new_config)
+
+	for hid, sids in old_config.getHomeSensors().items():
+		print("{} : ".format(hid), end="")
+		if hid in new_config.getHomeSensors():
+			new_sids = new_config.getHomeSensors()[hid]
+			if set(sids) == set(new_sids):
+				print("Same sensor ids")
+			else:
+				print("New sensors ids : ")
+				# sensors ids from new config not present in the other config
+				print([sid for sid in new_sids if sid not in sids])
+		else:
+			print("New home")
+
```

Comment:
Why not start the script to recompute data when the configuration is new ? We could forbid the case where we only insert the configuration without checking ?

Reply:
We could indeed run this script before inserting new configuration in the database and allow the insertion if there are changes, but it is not implemented yet.